packages/RichTextEditing-Core.package/RichTextEditorModel.class/instance/positionInTextAtIndex..st

```diff
@@ -0,0 +1,14 @@
+document information
+positionInTextAtIndex: anIndex
+
+	| lineCounter currentLine positionInLine |
+
+	lineCounter := 0.
+	currentLine := self displayedLines 
+		detect: 
```

Comment:
Isn't this the collection count function?

Reply:
It actually returns the first element for which the block evaluates to true